debezium-storage/debezium-storage-redis/src/main/java/io/debezium/storage/redis/offset/RedisOffsetBackingStore.java

```diff
@@ -67,7 +93,7 @@ public synchronized void stop() {
     /**
     * Load offsets from Redis keys
     */
-    private void load() {
+    public void load() {
```

Comment:
Please either change it to package visibility or mark it as ` @VisibleForTesting`.

Reply:
Done